### DISCUSSION TITLE
Looking for codes for Ferguson AV660

### DISCUSSION THREAD

**Discussion by orestesgaolin** [probonopd/irdb#177] [Code requests] [answered]
Hi there, I noticed there's no Ferguson codes at all as this is relatively small brand in Europe. I would appreciate somebody contributing codes for Ferguson amplifier AV660. I know that some physical universal remotes can support it, but i don't own one unfortunately.

**Comment by bengtmartensson** [comment, marked as answer]
[GlobalCache Control Tower](https://irdb.globalcache.com/) has them. But note their terms of conditions: you are allowed to use it only if you have their products (GlobalCache) in your net. In particular, you are not allowed to upload their data to other data bases, like the current one.

> I noticed there's no Ferguson codes at all as this is relatively small brand in Europe.

AFAIK, they are pretty bit in tractors...

**Comment by probonopd** [comment]
What you may do, though, is tell us which protocol, device, and functions the device uses. :-)

**Comment by orestesgaolin** [comment]
I requested the GlobalCache then.

> What you may do, though, is tell us which protocol, device, and functions the device uses. :-)

On one forum I read that Ferguson amplifiers may be rebranded JVC or Thomson devices. I also found this [files repository](https://ferguson-digital.eu/download/) and it contains the [manual for AV-660](https://ferguson-digital.eu/download/?dir=audio/amplituners/AV-660/manuals) and some specs about their [universal remotes](https://ferguson-digital.eu/download/?dir=rcu/RCU660) - not sure if this is helpful in any way.

The amplifier has some basic functions like power on/off, input sources like AUX, TAPE, AV1-3.

> AFAIK, they are pretty bit in tractors...

It seems like they are separate companies :P [Ferguson Digital seems to be Polish company](https://ferguson-digital.eu/?lang=en).
